cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/system/CDAPLogAppender.java

```diff
@@ -54,6 +54,8 @@ public class CDAPLogAppender extends AppenderBase<ILoggingEvent> implements Flus
                                                                      Constants.Logging.TAG_USER_SERVICE_ID,
                                                                      Constants.Logging.TAG_WORKER_ID,
                                                                      Constants.Logging.TAG_WORKFLOW_ID);
+  private static final int FOLDER_CLEANUP_BATCH_SIZE = 100000;
```

Comment:
Seems like this is only used inside `LogCleaner`? It should be defined in that class.

Reply:
Moved to LogCleaner